Lib/Inc/ICommand.h

```diff
@@ -0,0 +1,69 @@
+/**
+ * @file ICommand.h
+ *
+ */
+
+/*
+ * ICommand.h
+ *
+ *  Created on: Apr 26, 2022
+ *      Author: salavat.magazov
+ */
+
+#ifndef ICOMMAND_H_
+#define ICOMMAND_H_
+
+#include "../etl/string.h"
+
+#define MAX_COMMAND_SIZE   100
+#define MAX_ARGUMENT_COUNT 10
+
+class ICommand
+{
+public:
+    ICommand();
+
+    typedef enum STATUS_T
+    {
+        COMMAND_OK = 0,
+        ERROR_INVALID_NAME,
```

Comment:
what makes a name invalid? 
I thought this was for the controller to decide, if I the parser gets:
```
>setTem;erature(arg1,arg2);
```
Will this set this status?